docs/resources/service_level_objective.md

```diff
@@ -84,15 +84,15 @@ resource "datadog_service_level_objective" "bar" {
 ### Optional
 
 - `description` (String) A description of this service level objective.
-- `force_delete` (Boolean) A boolean indicating whether this monitor can be deleted even if it’s referenced by other resources (e.g. dashboards).
+- `force_delete` (Boolean) A boolean indicating whether this monitor can be deleted even if it's referenced by other resources (e.g. dashboards).
 - `groups` (Set of String) A static set of groups to filter monitor-based SLOs
 - `monitor_ids` (Set of Number) A static set of monitor IDs to use as part of the SLO
 - `query` (Block List, Max: 1) The metric query of good / total events (see [below for nested schema](#nestedblock--query))
 - `tags` (Set of String) A list of tags to associate with your service level objective. This can help you categorize and filter service level objectives in the service level objectives page of the UI. Note: it's not currently possible to filter by these tags when querying via the API
-- `target_threshold` (Number) The objective's target in `(0,100)`.
-- `timeframe` (String) The time frame for the objective. The mapping from these types to the types found in the Datadog Web UI can be found in the Datadog API documentation page. Valid values are `7d`, `30d`, `90d`, `custom`.
+- `target_threshold` (Number) The objective's target in `(0,100)`. This must match the corresponding thresholds of the primary time frame
```

Comment:
```suggestion
- `target_threshold` (Number) The objective's target in `(0,100)`. This must match the corresponding thresholds of the primary time frame.
```

Reply:
Fixed in [f421d46](https://github.com/DataDog/terraform-provider-datadog/pull/1774/commits/f421d461db0f71dfc3420048e70cf560ee979b8d)